pkg/validationfile/blocks/expectedrelations.go

```diff
@@ -74,9 +76,9 @@ func (ors *ObjectRelation) UnmarshalYAML(node *yamlv3.Node) error {
 }
 
 var (
-	vsSubjectRegex               = regexp.MustCompile(`(.*?)\[(?P<user_str>.*)\](.*?)`)
-	vsObjectAndRelationRegex     = regexp.MustCompile(`(.*?)<(?P<onr_str>[^\>]+)>(.*?)`)
-	vsSubjectWithExceptionsRegex = regexp.MustCompile(`^(.+)\s*-\s*\{([^\}]+)\}$`)
+	vsSubjectRegex                       = regexp.MustCompile(`(.*?)\[(?P<user_str>.*)\](.*?)`)
+	vsObjectAndRelationRegex             = regexp.MustCompile(`(.*?)<(?P<onr_str>[^\>]+)>(.*?)`)
+	vsSubjectWithExceptionsOrCaveatRegex = regexp.MustCompile(`^(?P<subject_onr>[^\]\s]+)(?P<caveat>\[\.\.\.\])?(\s+-\s+\{(?P<exceptions>[^\}]+)\})?$`)
```

Comment:
Because it didn't read as nice in my opinion:

```
[user:tom?] is <whatevre>
```

If you strongly prefer `?`, I can switch to it

Reply:
In either case it's not going to be obvious to the reader what it means, but the difference is that `...` already had some meaning associated and it would be misleading to folks that are familiar with ellipsis as a means to denote the absence of relation. Not feeling strongly, just surfaced the friction it caused me while reading it.